src/service/signup.ts

```diff
@@ -0,0 +1,24 @@
+import User from "../models/user";
+import bcrypt from "bcrypt";
+
+interface UserData {
+    name: string;
+    email: string;
+    password: string;
+}
+
+const createUser = async (userData: UserData): Promise<any> => {
+    const { name, email, password } = userData;
+    const hashPassword: string = await bcrypt.hash(password, 10);
```

Comment:
check for duplicate emails to see if the user already exists.

Reply:
I checked it and there won't be any duplicate emails.